### PR TITLE
Add `xo` as a linting tool

### PR DESCRIPTION
**Pull request by jfmengels**
This adds XO as a linting tool to have a consistent linter and style accross the Zeit projects, as requested by @leo.

### CODE REVIEW

**Comment by leo** [comment]
Looks great! But I think the `esnext` property is missing from the XO config 😊

File: index.js [hunk 1]
```diff
@@ -45,47 +44,53 @@ module.exports = function(val, options){
...
+      return n * s
     case 'milliseconds':
     case 'millisecond':
     case 'msecs':
     case 'msec':
     case 'ms':
-      return n;
+      return n
+    default:
+      return undefined
```
Comment (jfmengels) [inline index.js]: I had to add this code, which keeps the current behavior, but you might want to add a different behavior somehow (return NaN or something?)

**Comment by jfmengels** [comment]
Sorry, forgot to mention that.
I didn't add it on purpose, as I thought you might not want to make a breaking change. This package (probably?) works with older versions of Node, and ES2015-ifying it would necessarily be a breaking change and would limit its use to newer Node versions.

I'm fine with adding the `esnext` property though. Just let me know :)

**Comment by leo** [comment]
@jfmengels Ahhh, you're right. We're not transpiring anything atm, so we also shouldn't add the property!

File: index.js (hunk 1 above)
Comment (leo) [inline index.js]: Why should we add a different behavior (instead of returning `undefined`)?

Comment (leo) [inline index.js]: Got it! Let's keep this for now.

**Comment by jfmengels** [comment]
Will probably do a few more repos later today/this week-end. Let me know if you need help for the `zeit/now` repo too ;)

**Comment by leo** [comment]
@jfmengels Cool, thanks! 😊 Yea, I definitely need help there... What's the best way to handle that? Can you copy my branch and work from there?

**Comment by leo** [comment]
Oh, we also need the badge! 😄

**Comment by jfmengels** [comment]
> Yea, I definitely need help there... What's the best way to handle that? Can you copy my branch and work from there?

Sure, I'll try that. And you're right, forgot the badge. Will make PRs for that ;)

**Comment by leo** [comment]
@jfmengels Perfect! I already added the badge to async-throttle and ms 😊

**Pull request merged**